dtfs-central-api/api-tests/sql-db-helper.ts

```diff
@@ -0,0 +1,63 @@
+import { AzureFileInfoEntity, UtilisationDataEntity, UtilisationReportEntity } from '@ukef/dtfs2-common';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+
+const initialize = async () => await SqlDbDataSource.initialize();
+
+type SqlTableName = 'UtilisationReport' | 'UtilisationData' | 'AzureFileInfo';
```

Comment:
There is a lot of logic that typeorm handles for us in terms of cascade updates, deletes, upserts etc. I have defined the helper functions in here to work on one entity at a time so that we don't try to "outsmart" typeorm and let it handle most of the logic